doc/walkthrough.rst

```diff
@@ -0,0 +1,109 @@
+Walkthrough
+===========
+
+This document walkthroughs a simple scenario with Commissaire.
+
+Configuring a ContainerManager (Optional)
+-----------------------------------------
+If you will be using OpenShift, OCP, or Kubernetes then configuring a ContainerManager is the first thing to do.
+This essentially will tell commissaire how to communicate with your ContainerManager. When a cluster is associated
+to this ContainerManager new hosts will be automatically added into the the ContainerManager as nodes.
+
+Let's say you wanted to add a ContainerManager called ``ocp``,  which has a url of ``https://openshift.example.com``,
+and uses a token of ``aaa`` for authentication:
+
+.. include:: examples/commctl_create_container_manager.rst
+
+Creating a Cluster
+------------------
+Clusters are groupings of hosts. These hosts are expected to be similar to each other in functionality. In other words,
+the configurations of hosts in a cluster should not differ. While the functionality provided by the hosts may differ
+the system itself should not. Take OpenShift nodes as an example. Some nodes may be providing database services,
```

Comment:
*technically* it's the pods that run on the nodes that do different things. But you can apply the node-labels system get that kind of behavior (a node running just specific things). I.e., using node-labels ensure any pods with specific tags only run on nodes with matching node labels.

Not sure what my suggestion would be here. Maybe something imparting this kind of message?

> Some nodes may be hosting pods running different workloads, such as database services,
> web applications, or a mixture. However, the underlying hosts themselves are configured to be OpenShift nodes and ...